packages/suite/src/support/messages.ts

```diff
@@ -2477,6 +2477,14 @@ export default defineMessages({
         defaultMessage: 'Change PIN',
         id: 'TR_DEVICE_SETTINGS_CHANGE_PIN_TITLE',
     },
+    TR_DEVICE_SETTINGS_HAPTIC_FEEDBACK_DESC: {
+        defaultMessage: "Enable haptic feedback for device interactions",
```

Comment:
"Turn on" instead of "Enable"?

Reply:
cc @Hermez-cz 

![image](https://github.com/trezor/trezor-suite/assets/152899911/636c5a27-783d-478f-9eb2-e165bd682d28)


I accept the copy of the survivor ;)

Reply:
Enable sounds better to me, turn on is more like turn on the vacuum cleaner.

Reply:
done